%YAML:1.0
---
Camera.fx: 6.1913385009765625e+02
Camera.fy: 6.1907653808593750e+02
Camera.cx: 3.2507214355468750e+02
Camera.cy: 2.3738647460937500e+02
Camera.k1: 0.
Camera.k2: 0.
Camera.p1: 0.
Camera.p2: 0.
Camera.k3: 0.
Camera.width: 640
Camera.height: 480
Camera.fps: 30.
Camera.bf: 3.0956691741943359e+01
Camera.RGB: 1
ThDepth: 40.
DepthMapFactor: 1000.
ORBextractor.nFeatures: 1000
ORBextractor.scaleFactor: 1.2000000000000000e+00
ORBextractor.nLevels: 8
ORBextractor.iniThFAST: 20
ORBextractor.minThFAST: 7
Viewer.KeyFrameSize: 5.0000000000000003e-02
Viewer.KeyFrameLineWidth: 1
Viewer.GraphLineWidth: 9.0000000000000002e-01
Viewer.PointSize: 2
Viewer.CameraSize: 8.0000000000000002e-02
Viewer.CameraLineWidth: 3
Viewer.ViewpointX: 0
Viewer.ViewpointY: -6.9999999999999996e-01
Viewer.ViewpointZ: -1.8000000000000000e+00
Viewer.ViewpointF: 500
DepthImgSavePath: "./Imgs/DepthImgs"
RGBImgSavePath: "./Imgs/RGBImgs"
